[Doc] update doc for resolve_issues_effectively (#24642)
Azure/azure-sdk-for-python: doc/dev/issues/resolve_issues_effectively.md

## Usage error (For users)
If you are not familiar with the SDK usage of a service, you can find relevant examples in [this repo][sample repo] in most cases.

> For some common errors, you can check [here](#Summary Of Common Errors).

Reviewer: ```suggestion
For some common errors, you can check [here](##Summary Of Common Errors).
```

> For some common errors, you can check [here](#Summary Of Common Errors).

Reviewer: please check the view 👍 
![image](https://user-images.githubusercontent.com/70930885/171531277-c9e523eb-50ee-4414-9c1c-1b0ee03c2b5a.png)

> For some common errors, you can check [here](##Summary Of Common Errors).

Reviewer: ```suggestion
For some common errors, you can check [here](## Summary Of Common Errors).
```

> For some common errors, you can check [here](## Summary Of Common Errors).

Reviewer: the view does not take effect
![image](https://user-images.githubusercontent.com/70930885/171072440-7266416a-3622-4a9f-8843-94c48ea50c37.png)

<hr/>

## Summary Of Common Errors

### Error from Track1 to Track2
If the code you use needs to set wait() function for the Long Running Operation to wait for the result, it should be the SDK of Track1. We have stopped maintenance at present.

> We strongly recommend that you update the SDK version. Then you will find that in the SDK of Track2, we all use the function name prefixed with `begin_` for LRO operations, and the result() method can be used to get the returned result.

Reviewer: post guidance link: https://devblogs.microsoft.com/azure-sdk/migrating-python-management-libraries/

> ### Error from Service

Reviewer: ```suggestion
### Possible Error from Service
```

> Since the python SDK is generated based on the [rest API][rest API], it will not deliberately change the returned results. So if you have any questions about the response result value of the request, please open the issue under the [rest issue][rest issue].

Reviewer: `So if you have any questions about the response result value of the request,`
it is better:
`So if you think the returned result is not as expected,`

### Error like (AttributeError: 'PipelineResponse' object has no attribute 'get')

> When this error occurs, you can check the version of `msrest`. Currently, for most SDKs, it needs to be >= 0.6.21. FYI: [SDK dependency][SDK dependency]

Reviewer: ```suggestion
When this error occurs, you can check the version of `msrest` and upgrade it to latest version.
```

> ### Error like (AttributeError: 'PipelineResponse' object has no attribute 'get')
> When this error occurs, you can check the version of `msrest`. Currently, for most SDKs, it needs to be >= 0.6.21. FYI: [SDK dependency][SDK dependency]
> 
> You could upgrade the dependency before using the SDK to check if the issue has been solved.

Reviewer: ```suggestion
### Error like (AttributeError: 'PipelineResponse' object has no attribute 'get')
When this error occurs, you can check the version of `msrest` and upgrade it to latest version then try again.
```

<hr/>

> 

Reviewer: `Summary of Common  Errors` shall be put behind `Resorve issue` part
